jdbc/README.md

```diff
@@ -112,24 +112,65 @@ bin/ycsb run jdbc -P workloads/workloada -P db.properties -cp mysql-connector-ja
+db.batchsize=1000                           # The number of rows to be batched before commit (or executeBatch() when jdbc.batchupdateapi=true)
 ```
 
 Please refer to https://github.com/brianfrankcooper/YCSB/wiki/Core-Properties for all other YCSB core properties.
 
 ## JDBC Parameter to Improve Insert Performance
 
 Some JDBC drivers support re-writing batched insert statements into multi-row insert statements. This technique can yield order of magnitude improvement in insert statement performance. To enable this feature:
-- **db.batchsize** must be greater than 0.  The magniute of the improvement can be adjusted by varying **batchsize**. Start with a small number and increase at small increments until diminishing return in the improvement is observed. 
+- **db.batchsize** must be greater than 0.  The magnitude of the improvement can be adjusted by varying **batchsize**. Start with a small number and increase at small increments until diminishing return in the improvement is observed. 
 - set **jdbc.batchupdateapi=true** to enable batching.
 - set JDBC driver specific connection parameter in **db.url** to enable the rewrite as shown in the examples below:
   * MySQL [rewriteBatchedStatements=true](https://dev.mysql.com/doc/connector-j/8.0/en/connector-j-reference-configuration-properties.html) with `db.url=jdbc:mysql://127.0.0.1:3306/ycsb?rewriteBatchedStatements=true`
   * Postgres [reWriteBatchedInserts=true](https://jdbc.postgresql.org/documentation/head/connect.html#connection-parameters) with `db.url=jdbc:postgresql://127.0.0.1:5432/ycsb?reWriteBatchedInserts=true`
+
+## Configuration Properties control for database specific SQL syntax
+
+Each SQL statement may require database specific variation.  The variation is auto detected by examining `db.url`.  Optionally, `db.dialect` `configuration properties` can be used for supported databases.
+
+`db.url` is examined for presence of the following keywords: `oracle`, `phoenix`, `postgres`, `sqlserver`.  Blank means the default SQL is used.
+
+| Database 		| insert 	| select 	| delete 	| update 	| select ... limit 			|
+| --            | --      	| --		| --		| --		| --						|
+| Oracle        |       	|			|			|			| `FETCH FIRST ? ROWS ONLY`	|
+| Postgres      |       	|			|			|			| `FETCH FIRST ? ROWS ONLY`	|
+| Phoenix       | `upsert`  |			|			| `upsert`	|							|
+| SQL Server    |       	|			|			|			| `SELECT TOP (?)`			|
+
+
+`db.dialect` is supported for the following databases where auto detection is not sufficient and/or additional control is warranted:
+
+| Option | Database 			| insert 	| select 						| delete 	| update 	| select ... limit 					|
+| --            	| --       | --      	| --							| --		| --		| --								|
+| `db.dialect=jdbc:cockroach[:{time_interval}]` | CockroachDB AOST  |       	| `select .. as of system time ..`|			|			| `select .. as of system time ..`	|
```

Comment:
why can't the `time_interval` come either from a normal jdbc url parameter or a YCSB property that the cockroach dbflavor knows to look at?

Reply:
adding a non standard `time_interval` in JDBC url did't seem like a good practice.  `db.dialect` YCSB property was added for this purpose and to add the optional `time_interval`.